server.js

```diff
@@ -2,10 +2,10 @@ const express = require('express');
 
 const app = express();
 
-const PORT = 3000;
+process.env.PORT = 3000;
```

Comment:
Это немного не так работает. process.env содержит в себе переменные окружения, эти переменные должны быть readonly. Обычно они указываются при запуске node, либо в отдельном .env файле. Я предлагаю изменить записать на такую
```suggestion
const PORT = process.env.PORT || 3000;
```